src/main/resources/com/google/api/codegen/php/sample_common.snip

```diff
@@ -0,0 +1,212 @@
+@snippet methodCallSampleCode(apiMethod)
+    @if apiMethod.isSingularRequestMethod
+        ${@apiMethod.apiVariableName}->{@apiMethod.name}(\
+            {@sampleMethodCallArgList(apiMethod.initCode)})
+    @else
+        ${@apiMethod.apiVariableName}->{@apiMethod.name}()
+    @end
+@end
+
+@snippet longRunningSampleCode(apiMethod)
```

Comment:
The paradigm I've been using in other languages is that each of these sample functions would also get the `sample.initCode` as a param. For in-code samples, each of the configured samples would thus result in a separate function call with a different `sample`'s `initCode` being passed in. For standalone sample, we only do this for one sample,  `apiMethod.samples.get(0)`.

I was confused by how you're getting correct baselines without passing `initCode` to these functions, but I think I see it now: you are doing the init code separately, so by the time you get to these functions, all the `initCode`-related set-up is already done so the the remaining function call does not need to know it. Is that correct?

If so, then that's totally fine (and in fact, may well be the way we switch all languages to, since this is one way to enable the responsive APIs Explorer samples later).

Reply:
Yes, that was how php snippets originally handle `initCode`. I barely changed that part.